packages/react/src/create-strict-api/index.ts

```diff
@@ -128,8 +128,9 @@ export interface CompiledAPI<TSchema extends CompiledSchemaShape> {
     TRequiredProperties extends {
       requiredProperties: TAllowedProperties;
       requiredPseudos: TAllowedPseudos;
-    } = never
-  >(): Internal$XCSSProp<TAllowedProperties, TAllowedPseudos, TSchema, TRequiredProperties>;
+    } = never,
+    CustomTSchema extends CompiledSchemaShape = TSchema
```

Comment:
Oh! Is this enabling custom values?

Reply:
Yeah, so in this example we only allow a specific background color: https://github.com/atlassian-labs/compiled/blob/a03690befd404f6d5c78e3b067be3309ddf78b3a/packages/react/src/create-strict-api/__tests__/index.test.tsx#L431